images/metalk8s-keepalived/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# The keepalived binary from the package is not build with JSON enabled
+FROM ${BASE_IMAGE}@sha256:${BASE_IMAGE_SHA256} AS build-step
+
+# Keepalived version to use
+ARG KEEPALIVED_VERSION
+
+WORKDIR /home/keepalived
+
+RUN apk add --no-cache make gcc curl autoconf automake musl-dev libnl3-dev libnftnl-dev openssl-dev \
+  && curl --fail -Lo keepalived.tar.gz https://github.com/acassen/keepalived/archive/refs/tags/v${KEEPALIVED_VERSION}.tar.gz \
+  && tar xvf keepalived.tar.gz && cd "keepalived-${KEEPALIVED_VERSION}" \
+  && ./autogen.sh \
+  && ./configure --enable-vrrp -enable-sha1 --enable-json \
+  && make && cp bin/keepalived /keepalived
+
+
+FROM ${BASE_IMAGE}@sha256:${BASE_IMAGE_SHA256}
+
+# Timestamp of the build, formatted as RFC3339
+ARG BUILD_DATE
+# Git revision o the tree at build time
+ARG VCS_REF
+# Version of the image
+ARG VERSION
+# Version of the project, e.g. `git describe --always --long --dirty --broken`
+ARG METALK8S_VERSION
+
+# These contain BUILD_DATE so should come 'late' for layer caching
+LABEL maintainer="squad-metalk8s@scality.com" \
+      # http://label-schema.org/rc1/
+      org.label-schema.build-date="$BUILD_DATE" \
+      org.label-schema.name="metalk8s-keepalived" \
+      org.label-schema.description="MetalK8s Keepalived container" \
+      org.label-schema.url="https://github.com/scality/metalk8s/" \
+      org.label-schema.vcs-url="https://github.com/scality/metalk8s.git" \
+      org.label-schema.vcs-ref="$VCS_REF" \
+      org.label-schema.vendor="Scality" \
+      org.label-schema.version="$VERSION" \
+      org.label-schema.schema-version="1.0" \
+      # https://github.com/opencontainers/image-spec/blob/master/annotations.md
+      org.opencontainers.image.created="$BUILD_DATE" \
+      org.opencontainers.image.authors="squad-metalk8s@scality.com" \
+      org.opencontainers.image.url="https://github.com/scality/metalk8s/" \
+      org.opencontainers.image.source="https://github.com/scality/metalk8s.git" \
+      org.opencontainers.image.version="$VERSION" \
+      org.opencontainers.image.revision="$VCS_REF" \
+      org.opencontainers.image.vendor="Scality" \
+      org.opencontainers.image.title="metalk8s-keepalived" \
+      org.opencontainers.image.description="MetalK8s Keepalived container" \
+      # https://docs.openshift.org/latest/creating_images/metadata.html
+      io.openshift.tags="metalk8s,keepalived" \
+      io.k8s.description="MetalK8s Keepalived container" \
+      # Various
+      com.scality.metalk8s.version="$METALK8S_VERSION"
+
+COPY --from=build-step /keepalived /usr/sbin/
+COPY keepalived.conf.j2 /etc/keepalived/
+COPY check-ingress.sh /etc/keepalived/
+
+COPY generate-config.py /
+COPY entrypoint.sh /
+
+RUN apk add --no-cache libnl3 libnftnl bash curl py3-jinja2 py3-yaml \
+  && addgroup -S keepalived_script \
+  && adduser -D -S -G keepalived_script keepalived_script
+
+ENTRYPOINT ["/entrypoint.sh"]
+CMD ["/etc/keepalived/keepalived-input.yaml"]
```

Comment:
Shall we try to run keepalived as a non root user ? Maybe NET_ADMIN capability is enough to do so (ref: https://github.com/gen2brain/keepalived_exporter/issues/5k)

Reply:
Right, I will give it a try (I think you not copied the right link, but it was this one https://github.com/gen2brain/keepalived_exporter/issues/14)

Reply:
Done https://github.com/scality/metalk8s/pull/3844/commits/ee9dcf8b7dd1ff5989791df47bfdc5202563970e